models/keywordModel.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const keywordSchema = new Schema(
+  {
+    keyword: {
+      type: String,
+      required: true,
+    },
+    ownerId: {
+      type: String,
+      required: true,
+    },
+    includedKeyword: {
+      type: Array,
+    },
+    excludedKeyword: {
+      type: Array,
+    },
```

Comment:
```suggestion
    includedKeyword: {
      type: Array,
      default: [],
    },
    excludedKeyword: {
      type: Array,
      default: [],
    },
```

위와 같이 default 값을 빈 배열로 둬서 따로 includedKeyword와 excludedKeyword가 전달되지 않았을 때 처리를 고려하면 좋을 것 같은데, 어떻게 생각하실까요?

Reply:
[mongoose 공식문서](https://mongoosejs.com/docs/schematypes.html)를 찾아봤는데, 

> Arrays are special because they implicitly have a default value of [] (empty array).

이미 암시적으로 기본값으로 [](빈배열)이 설정되어 있어서, 저희가 다시 명시하지 않아도 될 것 같아요!